src/widgets/block.rs

```diff
@@ -68,6 +147,9 @@ impl Padding {
         }
     }
 
+    /// Defines the [`top`](Padding::top) and [`bottom`](Padding::bottom) padding.
+    ///
+    /// This leaves [`left`](Padding::left) and [`right`](Padding::right) to `0`.
```

Comment:
Either:
```suggestion
This leaves [`left`](Padding::left) and [`right`](Padding::right) at `0`.
```suggestion
or
```
This sets [`left`](Padding::left) and [`right`](Padding::right) to `0`.
```